src/carousel/carousel.ts

```diff
@@ -53,26 +55,41 @@ export class NgbSlide {
   encapsulation: ViewEncapsulation.None,
   host: {
     'class': 'carousel slide',
-    '[style.display]': '"block"',
-    'tabIndex': '0',
-    '(keydown.arrowLeft)': 'keyboard && prev(NgbSlideEventSource.ARROW_LEFT)',
-    '(keydown.arrowRight)': 'keyboard && next(NgbSlideEventSource.ARROW_RIGHT)'
+    '[attr.tabindex]': 'keyboard && !showNavigationIndicators ? 0 : null',
```

Comment:
Why `null` and not `-1` here?

Reply:
`null` means the attribute is not there.
`-1` makes the element still focusable when clicking on it, even though it is not reachable through tab.

Reply:
> `null` means the attribute is not there.

Oh yes! Stupid me 🤷 !
Because it's a binding, this is the default angular behaviour. `null` will remove the attribute. I always forget about this.

Reply:
But I am thinking that it may actually be a good idea to allow focusing this element on click. Otherwise, depending on what is inside the carousel, it may happen that nothing is focused on click, and that could be confusing if the user then wants to interact with the carousel with the keyboard.

Reply:
I have changed to use `-1` instead of `null` to allow focusing the `ng-carousel` element when clicking on it.